test/unit/lbp-wrapper.js

```diff
@@ -0,0 +1,137 @@
+const {expect} = require('chai');
+const { ethers } = require("hardhat");
+const { parseEther } = ethers.utils;
+
+const init = require("../test-init.js");
+const balancer = require("../helpers/balancer.js");
+const tokens = require("../helpers/tokens.js");
+const {constants, BN} = require('@openzeppelin/test-helpers');
+
+const deploy = async () => {
+	const setup = await init.initialize(await ethers.getSigners());
+	
+	setup.vault = await balancer.Vault(setup);
+	
+	setup.lbpFactory = await balancer.LBPFactory(setup);
+	
+	setup.Lbp = balancer.Lbp(setup);
+
+    setup.LBPWrapper = await init.LBPWrapper(setup);
+
+	setup.tokenList = await tokens.ERC20TokenList(2, setup.roles.root);
+
+	return setup
+}
+
+function sortTokens(tokens) {
+	if (tokens[0].address > tokens[1].address) {
+		const temp = tokens[0];
+		tokens[0] = tokens[1];
+		tokens[1] = temp;
+	};
+
+	return tokens;
+};
+
+describe("Contract: LBPWrapper", async () => {
+	let setup;
+	let swapsEnabled;
+	let tokenAddresses;
+    let sortedTokens;
+	const NAME = "Test";
+	const SYMBOL = "TT";
+	let startTime = Date.now();
+	let endTime = startTime+100000;
+	let WEIGHTS = [parseEther('0.6').toString(), parseEther('0.4')];
+	const END_WEIGHTS = [parseEther('0.4').toString(), parseEther('0.6')];
+	const POOL_SWAP_FEE_PERCENTAGE = parseEther('0.01').toString();
+    const NEW_SWAP_FEE_PERCENTAGE = parseEther('0.02').toString();
+    let initUserData;
```

Comment:
I recommend you to install [prettier](https://prettier.io/docs/en/editors.html) for your code editor. It will take care of line indentation.